src/bug-filing/common/get-issue-details-html.ts

```diff
@@ -52,19 +49,13 @@ export const getIssueDetailsHtml: IssueDetailsGetter = (
     data: CreateIssueDetailsTextData,
 ): string => {
     const body =
-        buildBodySectionHtml(
-            'Issue Details',
-            buildIssueDetailsHtml(data.ruleResult.help, data.ruleResult.helpUrl, data.ruleResult.ruleId),
-        ) +
-        buildBodySectionHtml('Application', buildApplicationHtml(data.pageTitle, data.pageUrl)) +
+        buildBodySectionHtml('Issue', buildIssueDetailsHtml(data.ruleResult.help, data.ruleResult.helpUrl, data.ruleResult.ruleId)) +
+        buildBodySectionHtml('Target Application', buildApplicationHtml(data.pageTitle, data.pageUrl)) +
         buildBodySectionHtml('Element Path', escape(data.ruleResult.selector)) +
         buildBodySectionHtml('Snippet', buildSnippetHtml(data.ruleResult.html)) +
         buildBodySectionHtml('How to fix', buildHowToFixHtml(data.ruleResult.failureSummary)) +
-        buildBodySectionHtml(
-            'To reproduce',
-            `Use <a href="http://aka.ms/AccessibilityInsights">${title}</a> to investigate the issue details`,
-        ) +
         buildBodySectionHtml('Environment', escape(environmentInfo.browserSpec)) +
+        '<br>===<br>' +
```

Comment:
nit: This is the only specified `html` in this function. Does it make sense to also move it in a separate function that returns  the `<br>===<br>'`!

Reply:
I'm refactoring how we create the issue details string so this code will soon disappear.

Reply:
You still want this PR to go in, correct?